time start_time
time end_time
int8 num_prompt
---
int8 next_act
